dumper/app/src/main/resources/oracle-stats/native/table-types-dtl.sql

```diff
@@ -0,0 +1,89 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  D.con_id "ConId",
+  D.owner "Owner",
+  D.table_name "TableName",
+  D.partitioned "Partitioned",
+  D.iot_type "IotType",
+  D.nested "Nested",
+  D.temporary "Temporary",
+  D.secondary "Secondary",
+  D.cluster_name "ClusterName",
+  D.source "Source",
+  E.partitioning_type "PartitioningType",
+  E.subpartitioning_type "SubpartitioningType",
+  E.partition_count "PartitionCount",
+  F.count "SubpartitionCount"
+FROM (
+  SELECT
+    A.con_id,
+    A.owner,
+    A.table_name,
+    A.partitioned,
+    A.iot_type,
+    A.nested,
+    A.temporary,
+    A.secondary,
+    A.cluster_name,
+    'TAB' source
+  FROM cdb_tables A
+  WHERE A.owner <> 'SYSTEM' AND A.owner NOT LIKE '%SYS'
+  UNION ALL SELECT
```

Comment:
Optional: It looks like there is discrepancy in whether the query that follows the `UNION ALL` is added inside parentheses or not. Compare with:

https://github.com/google/dwh-migration-tools/blob/f3cd28ba5345ff0c8a8c19eface048772525776e/dumper/app/src/main/resources/oracle-stats/native/db-objects.sql#L29

I would suggest choosing just one style (either with parentheses or without) and applying it to all scripts.

Reply:
I usually avoid the parentheses in those queries if I see that they can be avoided. I'll try to remove the parentheses in db-objects on next change.